tests/integration/components/bs-modal-test.js

```diff
@@ -246,4 +246,65 @@ module('Integration | Component | bs-modal', function (hooks) {
 
     assert.dom('.modal').isVisible('Modal is visible again');
   });
+
+  testNotBS3('it animates opening the modal', async function (assert) {
+    this.set('open', false);
+    await render(hbs`<BsModal @open={{this.open}}>Hello world!</BsModal>`);
+
+    this.set('open', true);
+    await waitFor('.modal.show');
+    assert.dom('.modal').hasClass('show');
```

Comment:
This assertion is a bit meaningless, given the previous line. But I don't really care :)